jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -169,6 +169,10 @@ public void buildDocker()
           HelpfulSuggestions.forIncompatibleBaseImageJavaVesionForGradle(
               ex.getBaseImageMajorJavaVersion(), ex.getProjectMajorJavaVersion()),
           ex);
+
+    } catch (InvalidImageReferenceException ex) {
```

Comment:
At some point I think we should try reducing the copy-pasted exception handling between each of the tasks/goals.

Reply:
Yeah, that's a sound idea.